resolvers/post_organization_query/organization_post_pagination.js

```diff
@@ -0,0 +1,328 @@
+const Post = require('../../models/Post');
+
+const postsByOrganizationConnection = async (parent, args) => {
+  var sort = {};
```

Comment:
replace all **var** with **let**

Reply:
The file contains only a pagination query. So I don't think that the global scope due to var will be an issue.

Reply:
https://blog.usejournal.com/awesome-javascript-no-more-var-working-title-999428999994

Let is a best practice in the javascript world.